driver/razerkbd_driver.c

```diff
@@ -1748,6 +1870,23 @@ static int razer_raw_event(struct hid_device *hdev, struct hid_report *report, u
     return 0;
 }
 
+/**
+ * Set static hid-events translation map
+ *
+ * [BWE-Wheel] 'BlackWidow Elite' generates wheel-events for volume control spinner
+ */
+static int razer_kbd_input_mapping(struct hid_device *hdev, struct hid_input *hi, struct hid_field *field,
+                                   struct hid_usage *usage, unsigned long **bit, int *max)
+{
+
+    // [BWE-Wheel]
+    if (USB_DEVICE_ID_RAZER_BLACKWIDOW_ELITE == hdev->product && HID_TYPE_USBMOUSE == hdev->type && HID_GD_WHEEL == usage->hid) {
```

Comment:
Please rather do `if (variable == constant)` instead of the way it's currently

Reply:
Also can you check here for the Razer Huntsman Elite? See #803

Reply:
Dear **@z3ntu**, Yes, I can check this issue and think the problem is the same and after answering [question about multimedia buttons](https://github.com/openrazer/openrazer/issues/803#issuecomment-483223819) I try to implement that keyboard full support too.

Reply:
Dear *@z3ntu*, (variable == constant) is not a good practice - the code must be as issue persistent as possible and misprint *constant = variable* vs *variable == constant* will be detected at first compilation process, but *Legacy for the Future* - you are project owner so I will try to reverse if-expressions when has a spare time.